README.md

```diff
@@ -157,6 +157,12 @@ To load images into `egui` you can use the official [`egui_extras`](https://gith
 
 [`eframe`](https://github.com/emilk/egui/tree/master/crates/eframe) on the other hand has a lot of dependencies, including [`winit`](https://crates.io/crates/winit), [`image`](https://crates.io/crates/image), graphics crates, clipboard crates, etc,
 
+### build on Linux
+
+In order to build eframe on linux you need to install some components: pkg-config, glib, atk, gdk/gtk and pango
+
+`sudo apt install pkg-config libglib2.0-dev libpango1.0-dev libgtk-3-dev libatk1.0-dev`
```

Comment:
These instructions are more or less duplicate of those at
https://github.com/emilk/egui/blob/d856f7b3a58fc65a81ff9657f5c6e1ab462b7bb6/README.md#L85

Reply:
Aha, OK. I missed that, mainly because they were under the 'demo' heading. My bad